packages/create-snowpack-app/index.js

```diff
@@ -125,10 +127,14 @@ if (requiredVersion < 10) {
     `{"name": "my-csa-app"}`
   );
   try {
-    await execa("npm", ["install", template, "--ignore-scripts"], {
-      cwd: targetDirectory,
-      all: true,
-    });
+    await execa(
+      useYarn ? "yarn" : "npm",
+      [useYarn ? "add" : "install", template, "--ignore-scripts"],
```

Comment:
Noticed that `useYarn` didn’t apply here. Is this correct?

Reply:
talked in person, lets try to skip this if template === local path